feature/floor-map/src/test/java/io/github/droidkaigi/confsched2023/FloorMapScreenTest.kt

```diff
@@ -36,6 +36,8 @@ class FloorMapScreenTest {
         floorMapScreenRobot {
             setupScreenContent()
             checkScreenCapture()
+            setFloorLevelToGround()
+            checkScreenCapture()
```

Comment:
Thanks. Almost LGTM! But I would like to add what we want to check to the file name. So could you add test method and maybe in that test you don't have to take two screenshot.

Reply:
I extracted the test method https://github.com/DroidKaigi/conference-app-2023/pull/657/commits/88cfd0dd81201e66ff8dfe9ecd74314eec5e9958. Please let me know if it's not what you intended 👋🏼